templates/product/index.html.twig

```diff
@@ -42,4 +42,13 @@
     </table>
```

Comment:
Nie ma sensu dodawanie wyświetlania flash messages osobno w każdym widoku. To powinno znaleźć się w głównym widoku - tym, który rozszerzają wszystkie inne widoki